quantum/quantum_keycodes.h

```diff
@@ -503,6 +503,43 @@ enum quantum_keycodes {
     MAGIC_UNSWAP_CTL_GUI,
     MAGIC_TOGGLE_CTL_GUI,
 
+#ifdef JOYSTICK_ENABLE
```

Comment:
Lets not hid the buttons behind if-defs.  This way, they'll compile fine, but just not work.  
```suggestion
```

Reply:
change integrated in the last merge